snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/refset/SnomedRefSetType.java

```diff
@@ -34,8 +34,16 @@ public enum SnomedRefSetType {
 	 * Simple map type.
 	 * @see #SIMPLE_MAP_VALUE
 	 * @ordered
+	 * @deprecated
 	 */
 	SIMPLE_MAP(1, "SIMPLE_MAP", "SIMPLE_MAP"),
+	
+	/**
+	 * Simple map from SNOMEDCT type. 
+	 * @see #SIMPLE_MAP_FROM_SNOMEDCT_VALUE
+	 * @ordered
+	 */
+	SIMPLE_MAP_FROM_SNOMEDCT(1, "SIMPLE_MAP_FROM_SNOMEDCT", "SIMPLE_MAP_FROM_SNOMEDCT"),
```

Comment:
I think it would be great to change this enum to a basic enum literal instead of using the old EMF style format. Also, the index here is incorrect, please move it to the end of the list.